src/solve_dc_powerflow.jl

```diff
@@ -12,7 +12,7 @@ const PTDFPowerFlowData = PowerFlowData{
 }
 
 # ? change this to have a more detailed definition ?
-const vPTDFPowerFlowData = PowerFlowData{}
+const vPTDFPowerFlowData = PowerFlowData{<:Any, <:Any}
```

Comment:
we want to figure this out before merging and releasing

Reply:
yeah this is https://github.com/NREL-Sienna/PowerFlows.jl/issues/50

Reply:
Done, see https://github.com/NREL-Sienna/PowerFlows.jl/issues/50#issuecomment-2458075639